src/components/navbar/Navbar.js

```diff
@@ -21,7 +21,7 @@ export const Navbar = () => {
                     <img src="https://imgur.com/5LUwfu8.png" alt="logo" width="30%" />
                 </Link>
             </div>
-            <div style={{marginLeft: '-30%'}}>
+            <div style={{marginLeft: '-30em'}}>
```

Comment:
Ehhh nawet porządne % trzeba zamienić na em. Żadnego poszanowania dla świata XDD

Reply:
XDD Procenty są o tyle złe, że bardzo zależą od ekranu i to w złym tego słowa znaczeniu. em wydaje mi się są bardziej stabilne XDDD